lib/infrastructure/network/open_ai.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:http/http.dart' as http;
+
+class OpenAiService {
+  Future<String> submitPrompt(String prompt) async {
+    var url =
```

Comment:
Prefer using `final` instead of `var` when the variable is not reassigned afterward